experimental/packages/opentelemetry-sdk-metrics-base/src/aggregator/Histogram.ts

```diff
@@ -37,19 +37,33 @@ function createNewEmptyCheckpoint(boundaries: number[]): Histogram {
     },
     sum: 0,
     count: 0,
+    hasMinMax: false,
+    min: Infinity,
+    max: -1
   };
 }
 
+function hasMinMax(histogram: Histogram): boolean{
+  return histogram.hasMinMax;
+}
+
 export class HistogramAccumulation implements Accumulation {
   constructor(
     private readonly _boundaries: number[],
+    private _recordMinMax = true,
     private _current: Histogram = createNewEmptyCheckpoint(_boundaries)
   ) {}
 
   record(value: number): void {
     this._current.count += 1;
     this._current.sum += value;
 
+    if(this._recordMinMax) {
```

Comment:
```suggestion
    if (this._recordMinMax) {
```

Reply:
Done in [031219f](https://github.com/open-telemetry/opentelemetry-js/pull/3032/commits/031219f43405aea3e5ccc8b8d66b02ef98d0c350). Thanks for catching that! :slightly_smiling_face: